init.sls

```diff
@@ -10,15 +10,33 @@ duplicity_ppa_repo:
 
 duplicity:
   pkg.installed:
-    {% if 'ftp://' in pillar['duplicity']['backend'] %}
-    - pkgs: [duplicity, lftp]
-    {% elif 'boto3+s3' in pillar['duplicity']['backend'] %}
-    - pkgs: [duplicity, python3-boto3]
-    {% elif 's3' in pillar['duplicity']['backend'] %}
-    - pkgs: [duplicity, python3-boto]
-    {% elif 'scp://' in pillar['duplicity']['backend'] %}
-    - pkgs: [duplicity, python3-paramiko]
+    - pkgs:
+      - duplicity
+    {% if 'ftp://' in pillar['duplicity']['backend'] or 'ftp://' in pillar['duplicity']['multi_config'] %}
```

Comment:
Does this work when `duplicity::multi_config` is not existing? (didn't test, just want to make sure there are not errors here when not specifying. I suppose the additional test scenario for simple configuration (see below) will find out.